GLTFSDK/Inc/GLTFSDK/GLTFResourceReader.h

```diff
@@ -110,6 +110,8 @@ namespace Microsoft
                 return ReadAccessor<T>(gltfDocument, accessor);
             }
 
+            std::vector<float> ReadFloatData(const Document& gltfDocument, const Accessor& accessor) const;
```

Comment:
Bit of a nit but can you move the ReadFloatData declaration to be after all the ReadBinaryData declarations?